.github/workflows/ci-angular.yaml

```diff
@@ -0,0 +1,80 @@
+name: Angular CI
+
+on:
+  push:
+    branches: [ "master" ]
+env:
+  IMAGE_NAME: angular
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  
+jobs:
+  testing:
+     runs-on: ubuntu-latest
+     outputs:
+        get-job-id-test: ${{steps.job-id-test.outputs.get-job-id-test-out}}
```

Comment:
naming is important

so convert `get-job-id-test: ${{steps.job-id-test.outputs.get-job-id-test-out}}` into something any human can quickly get.

Reply:
I`ll take it into accoint, but now i`ve deleted this part of code, priviously it was used for API queries

Reply:
this comment was about the semantic naming not about implementation. Over time you will see that naming and code style/quality will either make you love working or hate maintaining it. SO short clean names are key. The naming you have now - is fine for the computer and hard for the human being. Love your fellow co-engineers so they can THANK YOU later!

Reply:
Fixed